### DISCUSSION TITLE
Is "command: sh -c "yarn db:migrate --env=production-ssl-disabled && yarn start --env=production-ssl-disabled" " no longer necessary inside docker-compose.yml?

### DISCUSSION THREAD

**Discussion by ovizii** [outline/outline#6661] [Self Hosting] [answered]
I've been running outline for a while, and today I was looking for something in the docs and compared my docker-compose.yml file to the one in the docs and noticed I still had this line in mine:

`    command: sh -c "yarn db:migrate --env=production-ssl-disabled && yarn start --env=production-ssl-disabled"`

Is that no longer necessary or is it already built into the Dockerfile?

**Comment by tommoor** [comment, marked as answer]
Yes, migrations are run on startup automatically now, though this can be disabled with the `--no-migrate` flag
